src/main/java/com/ekart/controller/ProductController.java

```diff
@@ -58,7 +61,7 @@
         return new ResponseEntity<Product>(product, new HttpHeaders(), HttpStatus.OK);
     }
 
-    @PostMapping
+    @PostMapping(path = "/add")
```

Comment:
only 1 unique path post mapping can be present. since for fetching products i have changed it to post method, the existing add product method should contain different path